integration/tests/upgrade_current_master_habdev.sh

```diff
@@ -15,6 +15,19 @@ Environment=CHEF_DEV_ENVIRONMENT=true
 EOF
 }
 
+do_deploy() {
+    #shellcheck disable=SC2154
+    chef-automate deploy config.toml \
+        --product automate \
+        --product automate-dev \
```

Comment:
this looks wrong. we shouldn't need to specify both. also, this test should not need automate-dev

Reply:
https://github.com/chef/automate/pull/3935 is the fix

Reply:
@jaym This is a direct cut and paste from the product-dev test; does it need to be fixed there as well?

I'll remove this and rebase with https://github.com/chef/automate/pull/3935.

Reply:
that one just needs to specify automate-dev and not both